third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -3691,9 +3696,16 @@ static void SendMessageToFrontend(const v8_inspector::StringView& message) {
   v8::HandleScope scope(isolate);
 
   v8::Local<v8::Context> context = isolate->GetCurrentContext();
-  v8::Local<v8::Value> arg = v8::String::NewFromTwoByte(isolate, message.characters16(),
+  v8::Local<v8::Value> arg;
+  if (message.is8Bit()) {
+    arg = v8::String::NewFromOneByte(isolate, message.characters8(),
```

Comment:
**[Irrelevant Comment]** I see! That's the `utf8` issue that you mentioned yesterday, I imagine 😄